mlflow/store/file_store.py

```diff
@@ -585,6 +585,21 @@ def set_tag(self, run_id, tag):
         # Don't add trailing newline
         write_to(tag_path, self._writeable_value(tag.value))
 
+    def delete_tag(self, run_id, key):
+        """
+        Delete a tag from a run.
+        :param run_id: ID of the run that contains the tag to delete
+        :param key: name of the tag to delete
+        """
```

Comment:
name of the tag -> Name of the tag

Reply:
"to delete" in each of the param description is sounds a bit redundant.